lib/delayed/backend/mongoid.rb

```diff
@@ -84,7 +85,7 @@ def reload(*args)
 
         # Hook method that is called after a new worker is forked
         def self.after_fork
-          if Gem::Version.create(::Mongoid::VERSION) < Gem::Version.create('5.0.0')
```

Comment:
Doesn't say the same thing as the comment, which is it should only care about version 4.